tests/fixtures/scripts_project/pkg/__init__.py

```diff
@@ -59,3 +59,7 @@ class Scripts:
     class Deep:
         def fun(self):
             print("task!")
+
+
+def async_task(*args, **kwargs):
```

Comment:
Seems like theres a missing async here?

Reply:
yikes I committed the wrong version of the test! would you like to do the honors to fix it?

Reply:
I'd rather you did